hbase-client/src/test/java/org/apache/hadoop/hbase/client/TestMetricsConnection.java

```diff
@@ -150,51 +152,71 @@ public void testStaticMetrics() throws IOException {
 
     for (int i = 0; i < loop; i++) {
       METRICS.updateRpc(ClientService.getDescriptor().findMethodByName("Get"),
-        GetRequest.getDefaultInstance(), MetricsConnection.newCallStats(), false);
+        GetRequest.getDefaultInstance(), MetricsConnection.newCallStats(), null);
       METRICS.updateRpc(ClientService.getDescriptor().findMethodByName("Scan"),
-        ScanRequest.getDefaultInstance(), MetricsConnection.newCallStats(), false);
+        ScanRequest.getDefaultInstance(), MetricsConnection.newCallStats(),
+        new RemoteWithExtrasException("IOException", null, false, false));
```

Comment:
similar to other comment - is this possible, or would it always be
```suggestion
        new RemoteWithExtrasException("java.io.IOException", null, false, false));
```

Reply:
Good catch, it should be this form from server side response.